.github/workflows/test.yaml

```diff
@@ -14,6 +32,19 @@ jobs:
           toolchain: stable
           override: true
 
+      - name: Cargo fmt check
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all --check
+        
+      - name: Cargo check
+        uses: actions-rs/cargo@v1
+        timeout-minutes: 10
```

Comment:
We have timeout-minutes: 10 for the `test` step below because the float32 tests sometimes hang forever, we can remove it from `check`